phpBB/styles/prosilver/template/mcp_ban.html

```diff
@@ -13,10 +13,10 @@
 	<!-- BEGIN bans -->
 		ban_length['{bans.BAN_ID}'] = '{bans.A_LENGTH}';
 		<!-- IF bans.A_REASON -->
-			ban_reason['{bans.BAN_ID}'] = '{bans.A_REASON}';
+			ban_reason['{bans.BAN_ID}'] = '{{ bans.A_REASON | e('js') }}';
```

Comment:
We prefer no space before and after `|`:
```twig
{{ bans.A_REASON|e('js') }}
```

Reply:
Really? I think that the current way is actually more readable.